ruby/vampires.rb

```diff
@@ -0,0 +1,152 @@
+
+def vampires
+
+puts "Welcome to Immortal Life Technologies"
+puts""
+
+counter = 0
+puts "How many of the following questionnaires will be administered?"
+total_employees = gets.chomp.to_i
+	until counter == total_employees
+	puts "Please fill out the following questionnaire to better serve your needs."
+	puts ""
+		
+	puts "What was you full name given at the time of your birth?"
+	birth_name = gets.chomp.capitalize
+	puts ""
+		
+	puts "How many years have you roamed this planet?"
+	age = gets.chomp.to_i
+	puts ""
+
+	puts "What was the year of your birth?"
+	birth_year = gets.chomp.to_i
+	puts ""
+
+# Mortal Age Confirmation #
+
+	mortal_age = false
+	until mortal_age
+
+#	puts "How many years have you roamed this planet?"
+#	age = gets.chomp.to_i
+
+#	puts "What was the year of your birth?"
+#	birth_year = gets.chomp.to_i
+
+		current_year = 2017
+		if age == (current_year - birth_year)
+			mortal_age = true
+		else
+			mortal_age = false
+		end
+	end
+
+# Garlic Aversion Confirmation #
+
+	garlic_choice = nil
+	while garlic_choice == nil
+
+	puts "Today the cafeteria is serving spaghetti. Would you care for some garlic bread? (y/n)"
+	garlic = gets.chomp.downcase
+	puts ""
+
+		if garlic == "yes" || garlic == "y"
```

Comment:
I like you adding `garlic_choice = nil` at the end but the beginning of this can be simplified/refactored. If we assume that the user only entered yes or no answers we can simplify it to:
`garlic_choice = garlic == "yes" || garlic == "y"`
If the user entered anything but `"yes"` or `"y"` then `garlic_choice` would be set to `false`. This is because in Ruby the right side of an expression with the `=` operator evaluates first.